pkgs/applications/audio/dgedit/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchgit
+, pkgconfig
```

canonical name over alias
```suggestion
, pkg-config
```